CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * [CHANGE] Ingester: Creating label `native-histogram-sample` on the `cortex_discarded_samples_total` to keep track of discarded native histogram samples. #5289
 * [FEATURE] Store Gateway: Add `max_downloaded_bytes_per_request` to limit max bytes to download per store gateway request.
 * [FEATURE] Added 2 flags `-alertmanager.alertmanager-client.grpc-max-send-msg-size` and ` -alertmanager.alertmanager-client.grpc-max-recv-msg-size` to configure alert manager grpc client message size limits. #5338
+* [FEATURE] Query Frontend: Add `cortex_discarded_queries_total` metric for throttled queries. #5356
```

Comment:
I think `cortex_failed_queries_total` is a better name than `cortex_discarded_queries_total` based what we are doing in this PR.  It doesn't make sense that a query can be "discarded".

using "failed" has the benefit that we can use the same metric name for 5xx  later on -- just have different value for `reason`. 

I would recommend to change this entry to 
`[FEATURE] Query Frontend: Add `cortex_failed_queries_total` metric for failed queries with `reason` label.  #5356`

Reply:
I agree `cortex_discarded_queries_total` is not a good name here. What I want to track is probably `cortex_throttled_queries_total`.
`cortex_failed_queries_total` is too general and I don't really want to track 5xx with this metric, 5xx can be of difference reasons and I feel it is more useful to track throttled queries for now.

Reply:
I see, yea `cortex_throttled_queries_total` make more sense. But "throttled" in this context is still weird because it's supposed to mean "flow control", but what we are doing here is not flow control when chunk is too big. 


How about `cortex_rejected_queries_total`?

Reply:
Love the name of `cortex_rejected_queries_total`!